segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/Throttler.java

```diff
@@ -122,6 +122,8 @@ boolean isThrottlingRequired() {
                 if (remaining > 0 && remaining < delay.get().getDurationMillis()) {
                     delay.set(delay.get().withNewDelay(remaining));
                 }
+                int incurredDelay = (int) (existingDelay.remaining.getInitial().toMillis() - existingDelay.remaining.getRemaining().toMillis());
```

Comment:
please add a method called `getElapsed` on `TimeoutTimer` that does this calculation. Doing so will remove the need to expose `getInitial` to the outside.